src/io.d

```diff
@@ -149,6 +176,19 @@ unittest
     assert(position.toString == expected);
 }
 
+@("convert position to offset")
+unittest
+{
+    import std.string : outdent, strip;
+
+    const position = Position("äöü.txt", 0, 43, "äöü or empty");
```

Comment:
Keep "äöü", to be as close as possible to the previous test.
Or use "don't care".